configs/default/IFRC-IFLIGHT_F745_AIO.config

```diff
@@ -108,26 +118,40 @@ dma pin A03 0
 # feature
 feature -RX_PARALLEL_PWM
 feature RX_SERIAL
+feature TELEMETRY
 feature LED_STRIP
 feature OSD
+feature ESC_SENSOR
 
 # serial
-serial 2 64 115200 57600 0 115200
+serial 0 8192 115200 57600 0 115200
+serial 1 64 115200 57600 0 115200
+serial 5 1024 115200 57600 0 115200
+serial 6 2 115200 57600 0 115200
 
 # led
 led 0 6,11::CTOBVIW:0
 led 1 7,11::CTOBVIW:0
 led 2 8,11::CTOBVIW:0
 led 3 9,11::CTOBVIW:0
 
+# aux
+aux 0 0 0 900 1300 0 0
+
 # master
 set baro_bustype = I2C
 set baro_i2c_device = 1
-set baro_hardware = BMP280
+set baro_hardware = AUTO
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set mag_hardware = AUTO
+set align_mag = CW180
```

Comment:
Does this board actually have an on-board magnetometer? If not then there's no value in setting the alignment as you have no idea how the user will install the external device.

Reply:
This board actually have an on-board magnetometer. But I will remove align_mag flag.

Reply:
I suggested removing it if the board **does not** have an onboard sensor. If the board actually has an onboard magnetometer the **you should include the alignment** that's appropriate for sensor's placement on the board.